src/main/java/com/faforever/client/remote/FafServerAccessorImpl.java

```diff
@@ -247,15 +250,17 @@ protected Void call() throws Exception {
 
             logger.info("FAF server connection established");
             Platform.runLater(() -> connectionState.set(ConnectionState.CONNECTED));
+            reconnectService.resetConnectionFailures();
```

Comment:
This also belongs in the reconnect service (and could be also be triggered by events on the message bus?)

Reply:
as u please

Reply:
actually no, events are bad here as they are async and create the need for synchronization